benchmark/crameri_et_al/case_2/crameri_benchmark_2.prm

```diff
@@ -69,21 +69,24 @@ subsection Material model
     set Thermal conductivities = 4.7
     set Thermal expansivities = 4e-5
     set Viscosities = 1.e21, 1.e20, 1.e23
-    set Viscosity averaging scheme = Harmonic
+    set Viscosity averaging scheme = harmonic
   end
 end
 
 
 subsection Mesh refinement
   set Additional refinement times        =
-  set Initial adaptive refinement        = 3
+  set Initial adaptive refinement        = 4
```

Comment:
This is the place where you get a linear interpolation of the top surface, after the nodes are placed using the initial 6 global refinement steps.